### PR TITLE
Change FirstLineSmaller to ask for font size

### PR DESCRIPTION
**Pull request by himajin-no-tameiki**

### CODE REVIEW

File: src/me/fru1t/subedit/utils/Utils.kt [hunk 1]
```diff
@@ -45,4 +45,13 @@ object Utils {
...
+  fun askForFontSize(): Int {
+    println("Font size?")
+    val size = readLine()?.toIntOrNull()
+    if (size == null) {
+      println("Invalid input...")
+      return askForFontSize()
+    }
+    return size
+  }
 }
```
Comment (fru1tstand) [inline src/me/fru1t/subedit/utils/Utils.kt]: Please add newline at end of file

File: src/me/fru1t/subedit/utils/Utils.kt [hunk 2]
```diff
@@ -45,4 +45,13 @@ object Utils {
   }
 
   fun askYes(): Boolean = readLine()?.toLowerCase() == "y"
+  fun askForFontSize(): Int {
```
Comment (fru1tstand) [inline src/me/fru1t/subedit/utils/Utils.kt]: Maybe `askForInt()` to make it a bit more general and either move the println to the caller OR pass in the prompt as a parameter

File: src/me/fru1t/subedit/utils/Utils.kt [hunk 3]
```diff
@@ -45,4 +45,13 @@ object Utils {
   }
 
   fun askYes(): Boolean = readLine()?.toLowerCase() == "y"
```
Comment (fru1tstand) [inline src/me/fru1t/subedit/utils/Utils.kt:47]: Add a newline to separate this function from the next

Comment (himajin-no-tameiki) [reply src/me/fru1t/subedit/utils/Utils.kt:47]: Done

File: src/me/fru1t/subedit/utils/Utils.kt [hunk 4]
```diff
@@ -45,13 +45,14 @@ object Utils {
...
+  fun askForInt(message: String): Int {
+    println(message)
     val size = readLine()?.toIntOrNull()
     if (size == null) {
       println("Invalid input...")
-      return askForFontSize()
+      return askForInt(message)
     }
     return size
   }
```
Comment (himajin-no-tameiki) [inline src/me/fru1t/subedit/utils/Utils.kt]: Changed this to be a more generic function that takes a prompt message as a parameter.

File: src/me/fru1t/subedit/utils/Utils.kt [hunk 5]
```diff
@@ -45,13 +45,14 @@ object Utils {
...
-}
\ No newline at end of file
+}
```
Comment (himajin-no-tameiki) [inline src/me/fru1t/subedit/utils/Utils.kt]: Added a newline at the end

File: src/me/fru1t/subedit/FirstLineSmaller.kt [hunk 6]
```diff
@@ -22,7 +22,8 @@ object FirstLineSmaller {
     println("RUNNING: First line smaller")
 
     val file = Utils.askForFile() ?: return
-    val fontSize = Utils.askForFontSize()
+
+    val fontSize = Utils.askForInt("Font size?")
```
Comment (himajin-no-tameiki) [inline src/me/fru1t/subedit/FirstLineSmaller.kt]: Passing a message parameter because of the change of the function

**Review by himajin-no-tameiki — COMMENTED** [review]
Fixed all

**Comment by himajin-no-tameiki** [comment]
🥳

**Pull request merged**